doc/level1.rst

```diff
@@ -2,25 +2,32 @@ Autocrypt Level 1: Enabling encryption, avoiding annoyances
 ===========================================================
 
 Autocrypt makes it easy for people to encrypt email.  This document
-describes the basic capabilities required for a mail app to be
-Autocrypt-capable at Level 1.
-
-The design of Level 1 is driven by usability concerns and by the
-realities of incremental deployment. A user may mix both
-Autocrypt-enabled mail clients and traditional mail clients and we'd
-like to avoid annoyances like unexpectedly unreadable mails while also
-supporting users who want to explicitly turn on encryption.
-
-For ease of implementation and deployment, Level 1 focuses on the use
-of Autocrypt on a single device.  We intend to :doc:`support
-multi-device synchronization (and other features) as part of Level
-2<next-steps>`.  We want to keep Level 1 simple enough that it's easy
-for developers to adopt it so we can drive efforts from real-life
-experiences as soon as possible.
-
-Throughout this document, we refer to a mail app or Mail User Agent (MUA)
-as though it were only capable of controlling a single e-mail account
-(see :ref:`multiaccounts` for more detail).
+specifies the requirements for Autocrypt Level 1 capable mail apps.
+
+The design of Autocrypt Level 1 is driven by usability concerns, and
+the realities of incremental deployment.  First, an Autocrypt-capable
+client can coexist with transitional mail clients.  That is, if
+someone uses an Autocrypt-capable mail client on her Desktop, she can
+continue to use a traditional mail client on her phone; she doesn't
+need to also use an Autocrypt capable client on phone.  (In fact, only
+a single Level 1 capable mail client can be used per mail account.)
```

Comment:
Not necessarily. The Setup Message allows to have more then 1 Autocrypt MUA per mail account